package.json

```diff
@@ -70,6 +72,7 @@
     "jest": "^22.0.4",
     "jest-cli": "^22.0.4",
     "node-sass": "^4.7.2",
+    "node-sass-tilde-importer": "^1.0.1",
```

Comment:
Consider using [node-sass-package-importer](https://github.com/maoberlehner/node-sass-magic-importer/tree/master/packages/node-sass-package-importer)
I used a different feature from [node-sass-magic-importer](https://github.com/maoberlehner/node-sass-magic-importer) and the experience was great.

Reply:
The tilde import part works the same way, but the other features are interesting.